pom.xml

```diff
@@ -1545,6 +1557,7 @@
 		<module>hibernate-dialect</module>
 		<module>eclipselink-platform</module>
 		<module>jboss-integration</module>
+		<module>swagger-integration</module>
```

Comment:
io.swagger's dependencies conflicts with jboss-integration's dependencies(jackson, etc), this the reason I use a separate module at the moment.

 it's not for test, current it use a servlet to bootstrap the Swagger(scan the anootations, form the swagger.json to Swagger UI JS)